mne/time_frequency/tests/test_tfr.py

```diff
@@ -264,7 +276,12 @@ def test_tfr_multitaper():
                                             time_bandwidth=4.0, picks=picks)
     power_evoked = tfr_multitaper(epochs.average(), freqs=freqs,
                                   n_cycles=freqs / 2., time_bandwidth=4.0,
-                                  return_itc=False)
+                                  return_itc=False, average=False).average()
```

Comment:
I added this just to test the average option on tfr_multitaper. tfr_multitaper works the same way for evoked and epochs. I'll modify the test to it's tested with epochs input. Makes more sense indeed.